web/src/components/Header/Header.module.css

```diff
@@ -0,0 +1,27 @@
+.header {
+  width: 100%;
+  box-shadow: 0 2px 4px var(--color-grey);
+  fill-opacity: 0.6;
```

Comment:
This will have no effect, see the screen recording

[Screencast From 2025-05-22 12-20-38.webm](https://github.com/user-attachments/assets/1480eb84-b139-4ffd-b1a6-317549df6898)

this is because [fill-opacity](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/fill-opacity) is an attribute for svg's

What are you trying to do here? Change the opacity of the entire header or just the image?

in either case you can just use the `opacity` attribute